lib/action_tracker/matchers/have_action_tracker_matcher.rb

```diff
@@ -1,17 +1,29 @@
 RSpec::Matchers.define :have_action_tracker do
   match do |actual|
-    actual.respond_to? 'track_event'
+    actual.class.included_modules.include? ActionTracker::Concerns::Tracker
+    filter_exists?(controller, :before, :initialize_session)
+    filter_exists?(controller, :after, :conditional_track_event)
   end
 
   description do
-    "responds to #track_event."
+    "have ActionTracker."
   end
 
   failure_message do |actual|
-    "expected that #{actual} responded to #track_event, but it didn't."
+    "expected that #{actual} to have ActionTracker, but it didn't."
```

Comment:
O que acha de mudar a mensagem pra _"expected #{actual} to include ActionTracker, but it didn't."_ ou _"expected #{actual} to have ActionTracker, but it hasn't."_